### ISSUE TITLE
Inputs missing labels

### ISSUE THREAD

**Issue by anselmbradford** [ericgio/react-bootstrap-typeahead#724]
Run https://ericgio.github.io/react-bootstrap-typeahead/ through https://wave.webaim.org/ and you'll see there are many "Missing form label" errors. I am running into this same issue and can't clear it because it's flagging the `aria-hidden` input.

**Comment by ericgio** [comment]
Hey @anselmbradford thanks for the report. I believe the "Missing form label" issue refers to specifically to how the examples are structured, since they indeed lack labels in most cases. This shouldn't be a general issue, however, and you can associate your own label with the typeahead as needed. The `aria-hidden` attribute is applied to the hint input, not the main input, so that shouldn't be an issue either. I don't think there's a bug here, but if I'm overlooking something, please provide additional detail and/or repro steps. Thanks!

**Comment by anselmbradford** [comment]
@ericgio Thanks for the response. I may be missing something, but in the react app I'm working on you can, for example, an aria-label can be added to `<AsyncTypeahead … >` with `inputProps={{aria-label: 'My accessible label',}}`, but that ends up on the main input and not the hint input. I see that `aria-hidden` is applied to the hint input, yet WAVE still flags it as not having a label, which implies there is an accessibility bug here, or possibly in WAVE itself.

**Comment by ericgio** [comment]
That's correct, the hint input is intentionally non-interactive and the `aria-hidden` attribute should be hiding that element from screen readers entirely. If WAVE is flagging it as an error, then it seems like potential a bug with WAVE. Fwiw, I don't see any errors using axe devtools. Could you post a screenshot of the error you're seeing with as much detail as possible? Thank you!

**Comment by anselmbradford** [comment]
@ericgio I think I figured out the issue. As explained [here](https://wave.webaim.org/help#hidden) WAVE errs on the side of exposing potential issues, even in hidden elements. The error looks like it could be ignored in this case, though I wonder if it would be helpful to allow the user of the typeahead the option to add a label to the hidden input to clear the error (and any confusion!). What do you think of duplicating any `aria-label` added to the main input over to the hidden input as well?

**Comment by ericgio** [comment]
@anselmbradford: Thanks for doing the additional research and for the suggestions. This is the relevant piece for me:

> If WAVE has detected an error in an element that is hidden and that is never presented to the end user, or that is perhaps modified to address the error before being presented (something WAVE cannot detect), the WAVE error can be ignored

The hint input is intended to be a visual cue only, and is therefore hidden from assistive technologies using `aria-hidden`. I think you can safely ignore the WAVE warning as described above.
